mvc/src/main/java/nextstep/mvc/controller/tobe/AnnotationHandlerMapping.java

```diff
@@ -21,10 +30,54 @@ public AnnotationHandlerMapping(Object... basePackage) {
     }
 
     public void initialize() {
+        log.info("Start Initializing AnnotationHandlerMapping!");
+
+        Reflections reflections = new Reflections(basePackage);
+
+        Set<Class<?>> controllers = reflections.getTypesAnnotatedWith(Controller.class);
+
+        controllers.forEach(this::setHandlerExecutionsOf);
+
         log.info("Initialized AnnotationHandlerMapping!");
     }
 
+    private void setHandlerExecutionsOf(Class<?> controller) {
+        Method[] methods = controller.getDeclaredMethods();
+
+        Arrays.stream(methods).forEach(method -> setHandlerExecutionsOf(controller, method));
```

Comment:
```suggestion
        Arrays.stream(methods)
            .forEach(method -> setHandlerExecutionsOf(controller, method));
```

여기도 줄바꿈 해주면 좋을 것 같습니다 ~!